src/Microsoft.Diagnostics.Monitoring.Options/MonitorCapability.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.Options
+{
+    public class MonitorCapability
+    {
+        public const string Exceptions = "exceptions";
```

Comment:
IMO it's a bit odd for this class to have these constants. Let's move them to be owned by the classes the define each capability (e.g. `CallStacksCapabilityPostConfigureOptions` would define the `CallStacks` const)

Reply:
What did you have in mind for capabilities that don't have a `PostConfigureOptions` (i.e. `HttpEgress`)?

Reply:
We could have a separate static class holding these constants if that's easier, my larger point here is that it's odd for this class in particular to own it, especially since it's public